content/intro-to-storybook/react/kr/using-addons.md

```diff
@@ -0,0 +1,160 @@
+
+<div class="aside">
+<strong>사용 가능한 애드온 목록을 찾고 계신가요?</strong>
+<br/>
+😍 Storybook에서 공식 지원되며, 커뮤니티에서 적극적으로 지원되는 애드온 목록은 <a href="https://storybook.js.org/addons/addon-gallery/">여기</a>에서 보실 수 있습니다.
+</div>
+
+저희는 특정 사용 사례에 대하여 애드온이 어떻게 구성되고 사용되는지 모두 다 적을 수는 없었습니다. 지금은 Storybook의 에코시스템 내에서 가장 인기 있는 애드온인 [knobs](https://github.com/storybooks/storybook/tree/master/addons/knobs)를 함께 구현해 보도록 하겠습니다.
+
+## Knobs 설정하기
+
+Knobs는 코드가 필요 없는 제한된 환경에서 디자이너와 개발자가 컴포넌트를 실험하고 사용해 볼 수 있는 놀라운 리소스입니다! 기본적으로 사용자가 스토리의 컴포넌트로 전달된 props를 조작해 볼 수 있도록 동적으로 정의된 필드들을 제공하는 것입니다. 이것을 지금부터 구현해보겠습니다.
+
+<video autoPlay muted playsInline loop>
+  <source
+    src="/intro-to-storybook/addon-knobs-demo.mp4"
+    type="video/mp4"
+  />
+</video>
+
+### 설치
+
+먼저, 필수적인 디펜던시를 설치해야 합니다.
+
+```bash
+yarn add -D @storybook/addon-knobs
+```
+
+Knobs를 `.storybook/main.js` 파일 안에 등록해주세요.
+
+```javascript
+// .storybook/main.js
+
+module.exports = {
+  stories: ['../src/components/**/*.stories.js'],
+  addons: [
+    '@storybook/preset-create-react-app',
+    '@storybook/addon-actions',
+    '@storybook/addon-knobs',
+    '@storybook/addon-links',
+  ],
+};
+```
+
+<div class="aside">
+<strong>📝 애드온을 등록하는 순서는 매우 중요합니다!</strong>
+<br/>
+애드온을 등록하는 순서는 애드온 패널의 탭에 표시되는 순서를 결정합니다(탭에 나타나는 애드온인 경우).
+</div>
+
+이상입니다! 이제 스토리에 적용해 볼 차례입니다.
+
+### 사용 방법
+
+`Task` 컴포넌트에 객체 타입의 knob를 사용해보겠습니다.
+
+먼저, `withKnobs` 데코레이터(decorators)와 `object` knob를 `Task.stories.js`에 가져옵니다.
+
+```javascript
+// src/components/Task.stories.js
+
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import { withKnobs, object } from '@storybook/addon-knobs/react';
+```
+
+<div class="aside">
+  만약 TypeScript를 사용하고 계신다면, import 하실 때 약간의 조정이 필요합니다.
+  <code>import { withKnobs, object } from '@storybook/addon-knobs'</code>를 사용해주세요.
+</div>
+
+다음은, `Task.stories.js` 파일의 `default` export에서 `decorators`에 `withKnobs`를 추가해주세요.
+
+```javascript
+// src/components/Task.stories.js
+
+export default {
+  component: Task,
+  title: 'Task',
+  decorators: [withKnobs],
+  excludeStories: /.*Data$/,
+};
+```
+
+마지막으로, `object` knob를 "default" 스토리 안에 넣어주세요.
+
+```javascript
+// src/components/Task.stories.js
+
+export const Default = () => {
+  return <Task task={object('task', { ...taskData })} {...actionsData} />;
+};
+```
+
+이제 하단 창의 "Action Logger" 탭 옆에 새로운 "Knobs" 탭이 나타납니다.
+
+[여기](https://github.com/storybooks/storybook/tree/master/addons/knobs#object)에 설명된 것처럼, `object` knob는 라벨과 "default" 객체를 매개변수로 받아들입니다.
+라벨은 애드온 패널의 왼쪽에 표시됩니다. 전달한 객체는 편집 가능한 JSON blob으로 표시됩니다. 유효한 JSON 객체를 보내는 한, 컴포넌트가 객체에 전달된 데이터를 바탕으로 조정될 것입니다!
+
+## Storybook의 범위를 넓혀주는 애드온
+
+이제 Storybook이 뛰어난 [CDD 개발환경](https://blog.hichroma.com/component-driven-development-ce1109d56c8e)을 제공해줄 뿐만 아니라, 상호작용을 할 수 있는 문서 자료 또한 제공합니다. PropTypes는 훌륭하지만, 디자이너 또는 컴포넌트의 코드에 익숙하지 않은 사람도 knobs 애드온이 구현된 Storybook을 통해서라면 컴포넌트가 어떻게 작동하는지를 쉽게 이해하실 수 있습니다.
+
```

Comment:
@katepark1009 when you're able please adjust the link to "CDD" link to https://www.componentdriven.org/